Casks/h/hyper-connect.rb

```diff
@@ -0,0 +1,27 @@
+cask "hyper-connect" do
+  version "1.0.3"
+  sha256 :no_check
+
+  url "https://mijia-ios-adhoc.cnbj1.mi-fds.com/mijia-ios-adhoc/hyperconnect/HyperConnect.dmg",
+      verified: "mijia-ios-adhoc.cnbj1.mi-fds.com/mijia-ios-adhoc/hyperconnect/"
+  name "Hyper Connect"
+  name "小米互联服务"
+  desc "Cross-device interconnection service for Xiaomi ecosystem"
+  homepage "https://hyperos.mi.com/continuity"
+
```

Comment:
```suggestion

  livecheck do
    url "https://cn-device.interconnect.miui.com/package/mac/checkUpdate?version=0.0.0"
    strategy :json do |json|
      json.dig("data", "versionInfo", "version")
    end
  end

```

We can add a `livecheck` to retrieve the current version using the URL from the in-app updater. You can find these feeds using [Proxyman](https://github.com/ProxymanApp/Proxyman).

Reply:
Sorry, when rummaging in pagehome, I didn't find a livecheck available.

Reply:
@WarmEpoch No problem, you can just commit my suggestions. I have tested them locally.